server/routes/pages.js

```diff
@@ -39,6 +39,50 @@ module.exports = function(server)
         event.emit('render', req, res, options);
     });
 
+    //match requested url with the news data url
+    function findPost(url){
+       var formatted = url.replace('/post/','');
+        
+        for (var i = 0; i < config.news.length; i++) {
```

Comment:
This could be cleaned up by using a  forEach instead of referencing the array by index - https://developer.mozilla.org/en-US/docs/Web/JavaScript/Reference/Global_Objects/Array/forEach